app/views/lessons/previews/show.html.erb

```diff
@@ -7,15 +7,18 @@
       <p class="mb-6 text-gray-500">Paste markdown here to check how it will look on the website!</p>
 
       <div>
-        <div data-controller='tabs share-preview' data-share-preview-url-value="<%= lessons_preview_share_path(format: :turbo_stream) %>" data-tabs-index='0' data-tabs-active-class="text-gray-700 bg-gray-300/50 hover:bg-gray-300 dark:bg-gray-700/90" data-tabs-inactive-class="hidden">
+        <div data-controller='tabs share-preview clear-preview' data-share-preview-url-value="<%= lessons_preview_share_path(format: :turbo_stream) %>" data-tabs-index='0' data-tabs-active-class="text-gray-700 bg-gray-300/50 hover:bg-gray-300 dark:bg-gray-700/90" data-tabs-inactive-class="dark:hover:bg-gray-500" data-tabs-inactive-panel-class="hidden">
           <div class="flex justify-between">
             <div class="flex items-center mb-3">
-              <button data-action="tabs#change" class="text-gray-600 bg-gray-300/50 hover:text-gray-800 hover:bg-gray-200 dark:bg-gray-700/40 dark:text-gray-300 rounded-md border border-transparent px-3 py-1.5 font-medium cursor-pointer focus:outline-none" data-tabs-target='tab'>
+              <button data-action="tabs#change" class="text-gray-600 bg-gray-300/50 hover:text-gray-800 hover:bg-gray-200 dark:bg-gray-700/40 dark:text-gray-300 rounded-md border border-transparent px-3 py-1.5 font-medium cursor-pointer focus:outline-none" data-tabs-target='tab' data-clear-preview-target='write'>
                 Write
               </button>
-              <button data-action="tabs#change" class="ml-2 text-gray-600 hover:text-gray-800 hover:bg-gray-200 dark:bg-gray-700/40 dark:text-gray-300 rounded-md border border-transparent px-3 py-1.5 font-medium cursor-pointer focus:outline-none" data-tabs-target='tab'>
+              <button data-action="tabs#change" class="ml-2 text-gray-600 hover:text-gray-800 hover:bg-gray-200 dark:bg-gray-700/40 dark:text-gray-300 dark:hover:bg-gray-500 rounded-md border border-transparent px-3 py-1.5 font-medium cursor-pointer focus:outline-none" data-tabs-target='tab'>
                 Preview
               </button>
+              <button data-action="clear-preview#clear" class="ml-2 text-gray-600 hover:text-gray-800 hover:bg-gray-200 dark:bg-gray-700/40 dark:text-gray-300 dark:hover:bg-gray-500 rounded-md border border-transparent px-3 py-1.5 font-medium cursor-pointer focus:outline-none" data-clear-preview-target='button'>
```

Comment:
The clear button being styled the same as tabs may be confusing. Clear seems like it would be better as an action button like we have for share instead of another tab:

<img width="1082" alt="Screenshot 2024-02-03 at 16 55 07" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/44468213-8c27-4d4b-909f-39a7065c537f">

This feels more consistent to me, tabs always to the left, action buttons always to the right 👨‍🍳

Reply:
Makes sense!